src/schema/schema_worker.c

```diff
@@ -66,6 +66,37 @@ __schema_tiered_worker(WT_SESSION_IMPL *session, const char *uri,
     return (ret);
 }
 
+/*
+ * __schema_layered_worker_verify --
+ *     Run a schema worker operation (which is verification) on the stable table; ingest is not
+ *     supported currently.
+ */
+static int
+__schema_layered_worker_verify(WT_SESSION_IMPL *session, const char *uri,
+  int (*file_func)(WT_SESSION_IMPL *, const char *[]),
+  int (*name_func)(WT_SESSION_IMPL *, const char *, bool *), const char *cfg[], uint32_t open_flags)
+{
+    WT_DECL_RET;
+    WT_LAYERED_TABLE *layered;
+
+    WT_RET(__wt_session_get_dhandle(session, uri, NULL, NULL, open_flags));
+    layered = (WT_LAYERED_TABLE *)session->dhandle;
+
+    WT_ASSERT(session, layered->stable_uri != NULL);
+    WT_ASSERT(session, file_func == __wt_verify);
+
+    WT_WITHOUT_DHANDLE(session,
+      ret =
+        __wt_schema_worker(session, layered->stable_uri, file_func, name_func, cfg, open_flags));
+    WT_ERR(ret);
```

Comment:
I don't know if we should return here. For instance, if we have EBUSY, we may want to continue and verify the `ingest` still?

Reply:
That's a particularly good point! But I think it's more related to the weakening the current restrictions activity and should be considered separately. I think for now since we don't verify the ingest table at all, it's fine to just remove this `WT_ERR(ret)`. WDYT?